Makefile

```diff
@@ -102,6 +113,72 @@ run-docker:
 		-p 8001:8001 \
 		hypothesis/lms:$(DOCKER_TAG)
 
+.PHONY: run-docker-dev
+run-docker-dev:
+	docker run \
+		--name lms \
+		--net h_default \
+		-v /Users/hmstepanek/Documents/work/dev/lms/lms:/var/lib/lms/lms \
+		-it \
+		--entrypoint '/bin/sh' \
+		-e DATABASE_URL=postgresql://postgres@lmspostgres/postgres \
+		-e GOOGLE_CLIENT_ID \
+		-e GOOGLE_DEVELOPER_KEY \
+		-e GOOGLE_APP_ID \
+		-e H_API_URL_PRIVATE \
+		-e H_API_URL_PUBLIC \
+		-e H_AUTHORITY \
+		-e H_CLIENT_ID \
+		-e H_CLIENT_SECRET  \
+		-e H_JWT_CLIENT_ID \
+		-e H_JWT_CLIENT_SECRET \
+		-e JWT_SECRET \
+		-e LMS_SECRET \
+		-e RPC_ALLOWED_ORIGINS \
+		-e VIA_URL \
+		-e SESSION_COOKIE_SECRET \
+		-e OAUTH2_STATE_SECRET \
+		-e NEW_RELIC_LICENSE_KEY \
+        -e NEW_RELIC_APP_NAME="lms (dev)" \
+		-p 8001:8001 \
+		hypothesis/lms:$(DOCKER_TAG) \
+	    -c "newrelic-admin run-program gunicorn --paste conf/development.ini"
+
+.PHONY: run-docker-test
+run-docker-test:
+	docker run \
+		--name lms \
+		--net h_default \
+		-v /Users/hmstepanek/Documents/work/dev/lms/lms:/var/lib/lms/lms \
+		-v /Users/hmstepanek/Documents/work/dev/lms/tests:/var/lib/lms/tests \
+		-it \
+		--entrypoint '/bin/sh' \
+		-e DATABASE_URL=postgresql://postgres@lmspostgres/postgres \
+		-e TEST_DATABASE_URL=postgresql://postgres@lmspostgres/lms_test \
+		-e GOOGLE_CLIENT_ID \
+		-e GOOGLE_DEVELOPER_KEY \
+		-e GOOGLE_APP_ID \
+		-e H_API_URL_PRIVATE \
+		-e H_API_URL_PUBLIC \
+		-e H_AUTHORITY \
+		-e H_CLIENT_ID \
+		-e H_CLIENT_SECRET  \
+		-e H_JWT_CLIENT_ID \
+		-e H_JWT_CLIENT_SECRET \
+		-e JWT_SECRET=test_secret \
+		-e LMS_SECRET \
+		-e RPC_ALLOWED_ORIGINS \
+		-e VIA_URL=https://example.com/ \
+		-e SESSION_COOKIE_SECRET \
+		-e OAUTH2_STATE_SECRET \
+		-p 8001:8001 \
+		hypothesis/lms:$(DOCKER_TAG) \
+	    -c "pytest -v -p no:cacheprovider tests/lms"
+
+.PHONY: test-db
+test-db:
```

Comment:
Add the command for building the test database. I wonder if this just shouldn't be part of the test command.